tests/test_network.py

```diff
@@ -0,0 +1,37 @@
+from autowebcompat import network
+from keras import backend as K
+import math
+import numpy as np
+
+arr1 = np.array([[1, 1]], dtype=np.float32)
+arr2 = np.array([[0, 1]], dtype=np.float32)
+
+arr3 = np.array([[0, 0]], dtype=np.float32)
+arr4 = np.array([[3, 2]], dtype=np.float32)
+
+
+def test_eucledian_distance():
+    dist = network.euclidean_distance([arr1, arr2])
+    dist2 = network.euclidean_distance([arr3, arr4])
+    assert (K.eval(dist) == 1)
```

Comment:
Nit: this is a weird way to arrange the assertions. Put the assertion related to a `euclidean_distance` call right after the call itself.